crates/indexer/src/indexer/tailer.rs

```diff
@@ -165,12 +165,20 @@ impl Tailer {
 
     /// Get starting version from database. Starting version is defined as the first version that's either
     /// not successful or missing from the DB.
-    pub fn get_start_version(&self, processor_name: &String) -> Option<i64> {
+    pub fn get_start_version(
+        &self,
+        processor_name: &String,
+        is_backfill: Option<bool>,
+    ) -> Option<i64> {
         let mut conn = self
             .connection_pool
             .get()
             .expect("DB connection should be available to get starting version");
 
+        let mut lookback_versions: i64 = 1500000;
```

Comment:
You should really make this a configurable amount

Reply:
actually i'll just do it here so that we're not introducing a weird flag.